quickstart/101-resource-group/variables.tf

```diff
@@ -1,5 +1,6 @@
-variable "resource_group_name" {
-  default = "myResourceGroup"
+variable "resource_group_name_prefix" {
+  default       = "rg"
+  description   = "Prefix of the resource group name that's combined with a random number so name is unique in your Azure subscription."
```

Comment:
should generalize this to ... 'with a random **ID** so ' ... as it is not really a number if we go with GUID and ID will cover pet names if we go that direction.